forge-gui/res/cardsfolder/upcoming/lord_of_the_nazgul.txt

```diff
@@ -0,0 +1,12 @@
+Name:Lord of the Nazgûl
+ManaCost:3 U B
+Types:Legendary Creature Wraith Noble
+PT:4/4
+K:Flying
+S:Mode$ Continuous | Affected$ Creature.Wraith+YouCtrl | AddKeyword$ Protection from Ringbearers | Description$ Wraiths you control have protection from Ring-bearers.
+T:Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever you cast an instant or sorcery spell, create a 3/3 black Wraith creature token with menace.
+SVar:TrigToken:DB$ Token | TokenAmount$ 1 | TokenScript$ b_3_3_wraith_menace | TokenOwner$ You | SubAbility$ DBAnimateAll
+SVar:DBAnimateAll:DB$ AnimateAll | ValidCards$ Wraith.YouCtrl | Power$ 9 | Toughness$ 9 | ConditionPresent$ Card.Wraith+YouCtrl | ConditionCompare$ GE9 | SpellDescription$ Then if you control nine or more Wraiths, Wraiths you control have base power and toughness 9/9 until end of turn.
```

Comment:
I expect the trigger description to be incomplete ingame

Reply:
I couldn't see a description at all when they become 9/9. Should I shorten the description, or just eliminate it?

Reply:
you need to merge it into a single TriggerDescription